webhooks-extension/src/components/WebhookCreate/WebhookCreate.js

```diff
@@ -136,6 +136,25 @@ class WebhookCreatePage extends Component {
     }
   }
 
+  static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Can we use `componentDidUpdate` instead? Generally `getDerivedStateFromProps` isn't required or recommended. More detail: https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#when-to-use-derived-state

Reply:
ok ill see if it does the job!